docs/mscolab.rst

```diff
@@ -23,10 +23,10 @@ Protecting Login
 ~~~~~~~~~~~~~~~~
 The login to the MSColab server can be protected by an additional auth method.
 
-**mss_mscolab_auth.py**
-
  .. literalinclude:: samples/config/mscolab/mss_mscolab_auth.py.sample
 
+Copy the above file to the mslib/mscolab directory, rename it to mss_mscolab_auth.py and make the necessary changes in the file. 
+
```

Comment:
on an installed MSS that's not that easy. The prefered solution is to have it in the PYTHONPATH. e.g. you can have a CONFIG dir with all these pyscripts and add this to the PYTHONPATH env var. 
Similiar to the section above https://mss.readthedocs.io/en/stable/mscolab.html?highlight=PYTHONPATH#configuring-your-mscolab-server

it must be only in a path where python finds it. Usualy on server setup you add this path to the PYTHONPATH environment.

Reply:
Ah, got it. Thank you.